Admin-app/src/components/DropdownDoctor/index.js

```diff
@@ -148,10 +148,10 @@ function DropdownDoctor(props) {
             Object.values(doctorsList).map((doctor) => (
               <MenuItem
                 className="PATIENT__table__data"
-                value={doctor.id}
+                value={doctor.email}
                 disabled={isDoctorAtFullCapacity(doctor)}
               >
-                {doctor.name}
+                {`${doctor.firstName} ${doctor.lastName}`}
```

Comment:
Is this what we are displaying for the dropdown menu?

Reply:
Yeah, thats what we had before too.